src/App.tsx

```diff
@@ -1,14 +1,68 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState, useMemo } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { TodoFilterTypes } from './types/TodoFilterTypes';
+
+const getFilteredTodos = (
+  todos: Todo[],
+  query: string,
+  selectedFilter: string,
```

Comment:
Why it's `string`? You have `enum` for this value

Reply:
sorry!! forgot to change